digdag-core/src/main/java/io/digdag/core/log/LocalFileLogServerFactory.java

```diff
@@ -192,7 +207,14 @@ public void log(byte[] data, int off, int len)
             public void close()
             {
                 try {
-                    output.close();
+                    temporaryOutput.close();
+
+                    try (InputStream in = Files.newInputStream(temporaryPath);
+                         OutputStream conclusiveOutput = new GZIPOutputStream(Files.newOutputStream(conclusivePath, CREATE, APPEND), 16*1024)) {
+                        ByteStreams.copy(in, conclusiveOutput);
+                        conclusiveOutput.close();
```

Comment:
This stream should be closed by try-with-resources, so this line can be removed?